### PR TITLE
rephrase some things and fix examples

### PR DESCRIPTION
**Pull request by jaakkocsc**

### CODE REVIEW

**Review by atoivett — APPROVED** [review]
The changes look okay to me.

File: docs/data/sensitive-data/crypt4gh_client.md [hunk 1]
```diff
@@ -1,187 +1,252 @@
...
 
+### Step 1: Install the Crypt4GH encryption GUI tool
 
+Download the version specific to your operating system from the [GitHub repository](https://github.com/CSCfi/crypt4gh-gui/releases):  
 
+- [Windows](https://kannu.csc.fi/s/iDiNR5HdwtFrXCY)
+- [Mac](https://kannu.csc.fi/s/88MFCb4wNRt2mwb)
+- [Linux](https://kannu.csc.fi/s/NAgiSeS8mFXKnC4)
 
+Please check that the tool has been digitally signed by CSC - IT Center for Science. After the download, you can find the Crypt4GH application in your downloads folder.
```
Comment (blankdots) [inline docs/data/sensitive-data/crypt4gh_client.md:43]: we cannot state this for all OS, but only for Windows.

Comment (jaakkocsc) [reply docs/data/sensitive-data/crypt4gh_client.md:43]: Other OSes do not show verification dialog so the whole sentence is a moot point. But if a dialog appears, then it is necessary to verify the signature. Explaining this for every other step moves focus away from the instructions and quite some of our guides should then be written separately for macOS/Windows/Linux.

For simplicity, I'd leave this as is.

**Review by kkmattil — APPROVED** [review]
OK

**Pull request merged**